apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -1,5 +1,16 @@
 /* eslint-env browser,jquery */
-/* global Quill, he, MathJax, QuillMarkdown, showdown, filterXSS */
+/* global Quill, he, MathJax, QuillMarkdown, showdown, DOMPurify */
+
+const rtePurify = DOMPurify();
+const rtePurifyConfig = { SANITIZE_NAMED_PROPS: true };
+
+rtePurify.addHook('uponSanitizeElement', (node, data) => {
+  if (data.tagName === 'span' && node.classList.contains('ql-formula')) {
+    // Quill formulas don't need their SVG content in the sanitized version,
+    // as they are re-rendered upon loading.
+    node.innerText = `$${node.dataset.value}$`;
+  }
+});
```

Comment:
My rationale was to avoid having multiple instances of this object for multiple RTE elements in a page. I created a local function in 4d60980ce22344175ddf98291be0590d8fbd7a71 to create a local scope. I also added the custom formula embed to the local scope.

Reply:
Yay for IIFE! Thanks for doing that.